src/lib/Hydra/Plugin/InfluxDBNotification.pm

```diff
@@ -3,9 +3,8 @@ package Hydra::Plugin::InfluxDBNotification;
 use strict;
 use parent 'Hydra::Plugin';
 use HTTP::Request;
-# use JSON;
 use LWP::UserAgent;
-# use Hydra::Helper::CatalystUtils;
+use Sys::Hostname;
```

Comment:
In some other modules we're using `Sys::Hostname::Long`, would that be preferrable?

Reply:
Either works. `Sys::Hostname` is in the core, but I see that we also bring in `Sys::Hostname::Long`.